app/build.gradle

```diff
@@ -24,4 +24,5 @@ dependencies {
     testCompile 'junit:junit:4.12'
     compile 'com.android.support:appcompat-v7:23.2.1'
     compile 'com.android.support:support-v4:24.0.0-alpha1'
+    compile 'net.danlew:android.joda:2.9.4.1'
```

Comment:
I just checked and that danlew library is what Chris Stewart recommends: https://github.com/dlew/joda-time-android